main.cc

```diff
@@ -1440,7 +1440,17 @@ struct add_wiring_tool : tool
     }
 
     void get_description(char *str) override {
-        sprintf(str, "Place wiring type %u", type);
+        switch (type) {
+        case wire_type_power:
+            sprintf(str, "Place wiring type %s", "power");
```

Comment:
Do notice that these top 2 sprintfs are using static strings, do they need to be sprintfs?